snippets/Tooltip.md

```diff
@@ -0,0 +1,69 @@
+### Tooltip
+
+Renders a tooltip component.
+<br/><br/>
```

Comment:
HTML should be avoided in markup. If you need a new line please use the proper markup

Reply:
Removed.